docker-compose.yml

```
@@ -106,6 +106,7 @@ services:
       PHP_FPM_UID: ${PHP_FPM_UID-1000}
       PHP_FPM_GID: ${PHP_FPM_GID-1000}
       HOST_PATH: ${PWD-}/${LOCAL_DIR-src}
+      WP_CONFIG_PATH: ${WP_CONFIG_PATH-/var/www/wp-config.php}
```

```suggestion
      WP_CONFIG_PATH: ${/var/www/wp-config.php}
```

Since install.js is hardcoding the location, I think this configuration would make things more brittle.

---

Good. I amended your suggest to remove the `${…}` since this isn't referencing a variable now.